subsys/disk/disk_access_stm32_sdmmc.c

```diff
@@ -0,0 +1,401 @@
+/*
+ * Copyright (c) 2020 Amarula Solutions.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Please put following line here:
```
#define DT_DRV_COMPAT st_stm32_sdmmc
```
Then use DT macros that don't need compatible argument